config/locales/es-conferences.yml

```
@@ -0,0 +1,47 @@
+es:
+  activemodel:
+    attributes:
+      organization:
+        admin_terms_of_use_body: Cuerpo de los terminos de uso del panel de administración
+      conference:
+        available_slots: Plazas disponibles
+        end_date: Fecha de fin
+        location: Localización
+        main_logo: Logo principal
+        objectives: Objectivos
```

Just this translation:
```suggestion
        objectives: Objetivos
```